src/components/Table/InlineEditRow.js

```diff
@@ -4,10 +4,8 @@ import { Table } from 'patternfly-react';
 // HACK: DragableRow gives this error:
 //   Stateless function components cannot be given refs. Attempts to access this ref will fail.
 // eslint-disable-next-line react/prefer-stateless-function
-class InlineEditRow extends React.Component {
+export class InlineEditRow extends React.Component {
```

Comment:
What is the preferred approach? using default exports or exports?

Reply:
For our React components, we should use named exports (i.e. not default exports).

That's the convention we started with, so we should stick with it consistently.

`$ grep -r 'export default' src/components --include="*.js"`

One exception is the `EditableDraggableTable` component, which uses a HoC function:

```
export default DragDropContext(HTML5Backend)(EditableDraggableTable);
```

and I didn't like having neither `_EditableDraggableTable` for the actual component nor redefining `EditableDraggableTable` variable (ESLint will warn about that).

So unless it really hurts readability/complexity, I'd always favor named exports in modules representing our React components.